x-pack/plugins/security_solution/cypress/integration/users/user_details.spec.ts

```diff
@@ -24,13 +24,14 @@ describe('user details flyout', () => {
   before(() => {
     cleanKibana();
     login();
+  });
+
+  it('shows user detail flyout from alert table', () => {
```

Comment:
Just curious to know why this has been changed.

Reply:
I've noticed this test failed once due to some timeout of interaction with UI element. And because it happened in `before` hook, test wasn't retried. So, I decided to move it within a test, so test can be retried if it happens again